tests/test_validators.py

```diff
@@ -583,42 +594,64 @@ def test_fail_invalid_iterable(self):
         with pytest.raises(TypeError):
             v(None, a, [42])
 
-    def test_fail_invalid_member_and_iterable(self):
+    @pytest.mark.parametrize(
+        "member_validator", [instance_of(int), [always_pass, instance_of(int)]]
+    )
+    def test_fail_invalid_member_and_iterable(self, member_validator):
         """
         Raise iterable validator error if both the iterable
         and a member are invalid.
         """
-        member_validator = instance_of(int)
         iterable_validator = instance_of(tuple)
         v = deep_iterable(member_validator, iterable_validator)
         a = simple_attr("test")
         with pytest.raises(TypeError):
             v(None, a, [42, "42"])
 
-    def test_repr_member_only(self):
+    @pytest.mark.parametrize(
+        "member_validator", [instance_of(int), [always_pass, instance_of(int)]]
+    )
+    def test_repr_member_only(self, member_validator):
         """
         Returned validator has a useful `__repr__`
         when only member validator is set.
         """
-        member_validator = instance_of(int)
-        member_repr = "<instance_of validator for type <{type} 'int'>>".format(
-            type=TYPE
-        )
+        if isinstance(member_validator, list):
+            member_repr = (
+                "_AndValidator(_validators=[{func}, "
+                "<instance_of validator for type <{type} 'int'>>])"
+            ).format(func=repr(always_pass), type=TYPE)
+        else:
+            member_repr = (
+                "<instance_of validator for type <{type} 'int'>>".format(
+                    type=TYPE
+                )
+            )
         v = deep_iterable(member_validator)
         expected_repr = (
             "<deep_iterable validator for iterables of {member_repr}>"
         ).format(member_repr=member_repr)
         assert ((expected_repr)) == repr(v)
 
-    def test_repr_member_and_iterable(self):
+    @pytest.mark.parametrize(
+        "member_validator", [instance_of(int), [always_pass, instance_of(int)]]
+    )
+    def test_repr_member_and_iterable(self, member_validator):
         """
         Returned validator has a useful `__repr__` when both member
         and iterable validators are set.
         """
-        member_validator = instance_of(int)
-        member_repr = "<instance_of validator for type <{type} 'int'>>".format(
-            type=TYPE
-        )
+        if isinstance(member_validator, list):
+            member_repr = (
+                "_AndValidator(_validators=[{func}, "
+                "<instance_of validator for type <{type} 'int'>>])"
+            ).format(func=repr(always_pass), type=TYPE)
+        else:
+            member_repr = (
+                "<instance_of validator for type <{type} 'int'>>".format(
+                    type=TYPE
+                )
+            )
```

Comment:
This should be two test methods.

Reply:
I split them in my latest commit

Reply:
Did you forget to push? There's still an `isinstance`?

Reply:
Ahh, there were two tests that used `isinstance`, I thought you wanted only one to be split. I just split the other one too